workspaces/tests/account.rs

```diff
@@ -32,6 +32,8 @@ async fn test_subaccount_creation() -> anyhow::Result<()> {
         Some(&Value::String(sub.id().to_string()))
     );
 
+    fs::remove_file(savedir.join(format!("{}.json", sub.id())))?;
```

Comment:
What's that file?

Reply:
This file is created on previous lines. I`m realy not sure it makes much sense since it`s stored in ./target but it seems like one less unnecessary file